types/canonical.go

```diff
@@ -56,12 +56,14 @@ func CanonicalizeProposal(chainID string, proposal *cmtproto.Proposal) cmtproto.
 // relating to vote extensions.
 func CanonicalizeVote(chainID string, vote *cmtproto.Vote) cmtproto.CanonicalVote {
 	return cmtproto.CanonicalVote{
```

Comment:
Reading one previous comment from Jasmina, I realise we need to add the necessary checks to make sure PBTS is active. Because this code no longer works with BFT Time (even if we decided to leave the timestamps in the commit structure, they would no longer be signed, so any full node on the way can now tamper with its value)